lms/templates/dashboard/_dashboard_credit_info.html

```diff
@@ -19,61 +19,47 @@
                 )
             )}
         </p>
-    % if not credit_status["purchased"] and not credit_status["error"] :
-        <p class="message-copy credit-eligibility-msg">
-            ${_("You are now eligible for credit. <b>Congratulations!</b>")}
+    <div class="credit-action">
+        % if not credit_status["purchased"] and not credit_status["error"] :
```

Comment:
not critical, but: `not credit_status["error"]` is being checked repeatedly, would it be saner to move that condition up a level (surrounding all the conditional credit status messages) ?